JenkinsfileStressTests.groovy

```diff
@@ -57,7 +57,7 @@ pipeline {
                             sh "cd /srv && sudo btrfs subvolume snapshot bench-snapshot bench-running-docker"
                             sh 'docker run -d --name=cassandra -p 9042:9042 -v /srv/bench-running-docker/cassandra:/var/lib/cassandra cassandra:3.11.3'
                             sh 'docker run -d --name=elasticsearch -p 9200:9200 -v /srv/bench-running-docker/elasticsearch:/usr/share/elasticsearch/data/elasticsearch  --env "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:6.3.2'
-                            sh 'docker run -d --name=tika logicalspark/docker-tikaserver:1.19.1'
+                            sh 'docker run -d --name=tika linagora/docker-tikaserver:1.22'
```

Comment:
why do you need a linagora built image ?

Reply:
Because `logicalspark` published a 1.22 version but tagged as `latest`.

I asked for a 1.22 tag here https://github.com/LogicalSpark/docker-tikaserver/issues/21 but never got an answer.

Thus as we don't want to rely on `latest` as it might break our tests upon some push of some external party (relying on mutable image), we decided to tag the 1.22 ourselves and pushed it on the linagora repo.